tango/steps/__init__.py

```diff
@@ -5,5 +5,13 @@
 
 __all__ = ["DatasetRemixStep", "PrintStep"]
 
+from .complex_arithmetic import (
+    AdditionStep,
+    CosineStep,
+    ExponentiateStep,
+    MultiplyStep,
+    SineStep,
+    SubtractionStep,
+)
```

Comment:
I don't see these steps being useful outside of examples, so I'd prefer that we don't put them into the `tango` library itself because it adds overhead.

Reply:
They are the best example of step graphs larger than 3 steps we have right now, so I want to keep them. But maybe they should be part of tests instead.

Reply:
I moved it to https://github.com/allenai/tango-example.

Reply:
Personally I think we should keep that example in this repo. It's a good example and you put a lot of work into it. If we keep it here we can add it to the docs and it will be seen by a lot more users. We can also ensure that it's kept up-to-date. We already have the `examples/` folder for things like this.

Reply:
I brought it back in 23d78247376edf729a73d5eb02afea1537618ffa.